cmd/kubelet/app/options/container_runtime.go

```diff
@@ -29,6 +29,22 @@ const (
 	// When these values are updated, also update test/e2e/framework/util.go
 	defaultPodSandboxImageName    = "k8s.gcr.io/pause"
 	defaultPodSandboxImageVersion = "3.1"
+	defaultPodSandboxSeccomp      = `{
```

Comment:
Is this only used by docker? Maybe it should be moved to the dockershim package in that case?

Reply:
Currently yes, but once the CRI api is changed, this would also be used there.